README.md

```diff
@@ -221,11 +217,29 @@ If you want to clean out the navigation, articles, categories, and related media
 ./bin/clear-data.sh
 ```
 
-
 ### GATSBY!! (or the start of it)
+
 Gotta start somewhere! I'm putting a placeholder header section for now that will further outline how to work with support-docs on Gatsby.
 
+#### How to preview changes
+
+1. Create a branch
+2. Create a PR to master
+3. Go to https://www.gatsbyjs.com/dashboard/organization/60c30363-21fd-4739-a2ca-75f5011a3a1b/sites
```

Comment:
You might want to mention something about needing an account. Something like "contact the FE team if you need an account".